Bangkok Post.js

```diff
@@ -0,0 +1,175 @@
+{
+	"translatorID": "7f74d823-d2ba-481c-b717-8b12c90ed874",
+	"label": "Bangkok Post",
+	"creator": "Matt Mayer",
+	"target": "https://www.bangkokpost.com/.*",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-09-07 19:12:01"
+}
+
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 Matt Mayer
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	return 'newspaperArticle';
+}
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+function getMetaTag(doc,attr, value, contentattr) {
```

Comment:
Various formatting issues throughout like the missing space between `doc,` and `attr`. If possible, run `npm run lint -- Bangkok\ Post.js` on the file from the command line. (If you can't, don't worry about it — we can lint separately after.)

Reply:
lint now runs clean